tests/skills/slash_commands.test.js

```diff
@@ -26,12 +26,15 @@ const cases = require('../../commands/cases')
 jest.mock('../../commands/help_me')
 const help = require('../../commands/help_me')
 
+jest.mock('../../commands/find_clinic')
+const findClinic = require('../../commands/find_clinic')
+
 describe("slash_commands",()=>{
   let mockController = {}
   let mockBot = {
     replyPublic: function(){
       return 'replyPublic called'
-    }
+		}
```

Comment:
```suggestion
    }
```
this one is a nitpick lol

Reply:
not at all. I think I have tabs configured differently in my editor.